docker-hub/scim.md

```diff
@@ -1,67 +1,151 @@
+8. Click **Provision Azure Active Directory Users**, in **Attribute Mappings**, and keep the **userName**, **Active**, **givenName**, **familyName**. Delete the other attributes listed.
+
+    ![attribute-mapping](images/attribute-mapping.png){:width="700px"}
+
+9. Click **Start Provisioning** to begin the full synchronization.
+
+     > **Note**
+     >
+     > It can take up to 30 minutes to begin provisioning. You can also
+     > provision on demand provisioning with one user that is already assigned
+     > to the application.
+
+### Onelogin
+
+1. In Onelogin, navigate to **Applications** > **Applications** > **Add app**.
+2. In the search field, enter **SCIM Provisioner with SAML (SCIM vs Core)** and select the item in the results.
+3. Enter **Docker Hub** as the display name and click **Save**.
+4. Navigate to the left navigation, and select **Configuration**.
+5. In a separate tab, navigate to **Docker Hub** > **Settings** > **Security** > **SSO** > **SAML**.
+6. Copy the following fields from Docker Hub, in to OneLogin:
+
+    * Entity ID: SAML Audience URL
+    * ACS URL:  SAML Consumer URL
+    * SCIM Base URL: SCIM Base URL
+    * Custom Headers: 'Content-Type: application/scim+json'
+    'User-Agent: OneLogin SCIM'
+    * SCIM Bearer Token: SCIM Bearer Token
+    * SCIM JSON Template
+
+    ![application-details](images/application-details.png){:width="700px"}
+
+7. Click **API Connection** > **Enable** and **Save**.
+8. Navigate to the **Parameters** tab, click **scimusername**, select **Email** and **Save**.
+
+    ![parameters-tab](images/parameters-tab.png){:width="700px"}
+
+10. Navigate to the SSO tab, and copy the **SAML 2.0 Endpoint (HTTP)** url and paste it in to [**Docker Hub**](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} > **Settings** > **Security** > **SSO** > **SAML Sign-in URL**.
+11. In the **X.509 Certificate** field, click **View Details**.
+12. Copy the **PEM certificate**, and paste it in to [**Docker Hub**](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} > **Settings** > **Security** > **SSO** > **Key x509 Certificate** and click **Save**.
+12. In Onelogin, navigate to the **Access** tab, enable the app for `ol_role` that was already created and click **Save**.
+14. Navigate to the **Provisioning** tab and select **Enable provisioning**, and deselect the other options.
+15. In the drop-down, select **Suspend** and click **Save**.
+
+
+#### Generate a full-sync
+
+1. In Onelogin, navigate to **Applications** and select the Docker Hub application.
+2. In **Users**, click **Apply to All** and **Reapply Mappings**.
+3. Click **Confirm** to provision your users in Docker Hub.
+
+    > **Note**
+    >
+    > This creates a full-sync and any user that was not previously
+    > provisioned is now provisioned in Docker Hub.
+
+![scim-okta-button](images/scim-provisioner-saml.png){:width="700px"}
```

Comment:
suggestion: use this screenshot instead: https://www.notion.so/dockerinc/TC003-Configure-SCIM-on-OneLogin-e8aa012f521642718395260e91315105#6d8ee11bfc1c4948aa7e2e38669d93a6

Reply:
Updated